book/content/software_checklist/software_checklist.md

```diff
@@ -0,0 +1,49 @@
+# Checklist
+
+## Summary
+Here we provide a short checklist for software projects. The rest of this chapter elaborates on the various point in this list.
+
+## The list
+
+The __bare minimum__ that every software project should do, from the start, is:
+* Pick and include an [open source license](/licensing/01/softwarelicenses).
+* Use [version control](/version_control/version_control).
+* Use a [publicly accessible](/open_research/02/opensourcesoftware) version control repository.
+* Add [documentation describing the project](/documentation/documenation#Project_Documentation).
+
+We recommend that you also do the following (from the start of the project):
+* Use [code quality tools](/code_quality/code_quality).
+* [Testing](/testing/testing).
+
+Additional steps depend on the goal of the software (zero or more can apply):
+* [I'm publishing a paper](#im-publishing-a-paper).
+* [I'm expecting users](#im-expecting-users).
+* [I'm expecting contributors](#im-expecting-contributors).
+
+## I'm publishing a paper
+
+* [Make your software citable](/software_checklist/01/making_software_citable).
+* Cite DOI in paper (see the "[Further reading](#further-reading)" section at the end of the chapter).
+
+## I'm expecting users
+
+* [Release](/software_checklist/02/releases) your software.
+* Provide [user documentation](/documentation/documenation#Project_Documentation).
+* Provide [easy installation](/software_checklist/02/releases#one-command-install).
+* Provide issue tracker.
+
+## I'm expecting contributors
+
+* Provide [contributor guidelines](/documentation/documenation#CONTRIBUTING) and [development documentation](/documentation/documenation#Code_Documentation).
+* Provide a means of communication: i.e. use Github issues, mailing list, not private email.
+* Implement and add a [code of conduct](/collaborating_github/3/getting_contributors#code-of-conduct).
+* Add a [contribution guideline](/collaborating_github/3/getting_contributors#code-of-conduct).
```

Comment:
1. the contrition guidelines are above
2. why this link leads to code of conduct?

Reply:
done